serving/src/main/java/ai/djl/serving/Arguments.java

```diff
@@ -82,13 +83,19 @@ public Path getConfigFile() {
             configFile = System.getProperty("ai.djl.conf", null);
         }
         if (configFile != null) {
+
             Path file = Paths.get(configFile);
             if (!Files.isRegularFile(file)) {
                 throw new IllegalArgumentException("Configuration file not found: " + configFile);
             }
             return file;
         }
-        Path file = Paths.get("config.properties");
+        Path modelServerHome = Paths.get(ConfigManager.getModelServerHome());
+        Path file = modelServerHome.resolve("conf/config.properties");
+        if (Files.isRegularFile(file)) {
+            return file;
+        }
+        file = modelServerHome.resolve("conf/config.properties");
```

Comment:
what is difference between line `94` and `98` ?

Reply:
correct. @frankfliu I think the line 94 and 98 are repeated.